client/src/components/member/MemberForm.js

```diff
@@ -36,6 +37,8 @@ export default function MemberForm(props) {
   const [submitSuccessful, setSubmitSuccessful] = useState(null);
   const [promptMessage, setPromptMessage] = useState('');
 
+  const history = useHistory();
+
```

Comment:
Let's move this to line 26 where `classes` is defined, just so that similar things are defined together.